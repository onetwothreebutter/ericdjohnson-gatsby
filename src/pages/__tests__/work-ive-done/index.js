import React from 'react'
import {shallow} from 'enzyme'
import shallowToJson from 'enzyme-to-json'

import WorkIveDonePage from '../../work-ive-done/index'

test('Work Ive Done Page should render correctly', () => {
    const wrapper = shallow(
        <WorkIveDonePage/>
    );
    expect(shallowToJson(wrapper)).toMatchSnapshot();
});