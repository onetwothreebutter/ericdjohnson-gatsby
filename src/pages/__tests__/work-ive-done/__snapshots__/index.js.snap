// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Work Ive Done Page should render correctly 1`] = `
<div>
  <section
    className="work__banner"
  >
    <div
      className="work__page-width"
    >
      <h1
        className="site__banner-heading"
      >
        Work I've Done
      </h1>
      <PhotoCredit
        image_url="http://unsplash.com/@groosheck?utm_medium=referral&utm_campaign=photographer-credit&utm_content=creditBadge"
        name="MichaÅ‚ Grosicki"
      />
    </div>
  </section>
  <section
    className="work-tiles"
  >
    <WorkTile
      description="I love bringing the web to life!"
      heading="Web Animation"
      link="web-animations"
      svg={<UNDEFINED />}
    />
    <WorkTile
      description="My fun, reusable interface elements"
      heading="UI Components"
      link="ui-components"
    />
    <WorkTile
      description="A couple of examples of UX improvement ideas"
      heading="UX Design"
      link="ux-design"
    />
    <WorkTile
      description="Some of my bright ideas"
      heading="Product Design"
      link="product-design"
    />
    <WorkTile
      description="Stickers, t-shirts, and logos I've designed"
      heading="Visual Design"
      link="visual-design"
    />
    <WorkTile
      description="Good writing makes good code"
      heading="Writing"
      link="writing"
    />
  </section>
</div>
`;
